src/relay/pass/fuse_ops.cc

```diff
@@ -870,7 +872,7 @@ class FuseMutator : private ExprMutator {
       return MakeNewFunction(ret_group, tuple_get->checked_type(), new_node);
     }
     // This is an intermediate node in the group
-    return new_node;
+    return std::move(new_node);
   }
 
   Expr MakeNewFunction(GraphPartitioner::Group* group, Type ret_type, Expr body) {
```

Comment:
const &

Reply:
@MarisaKirisame which part do you want to make `const&`?

Reply:
Type/Expr.